Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -47,6 +48,10 @@ @interface WKInteractionRegion : NSObject
 @implementation WKInteractionRegion
 @end
 
+@interface UIView (IPI_RemoteLayerTreeInteractionRegionLayers)
+- (UIScrollView *)_scroller;
```

Comment:
Is there a path to make this SPI / eventually remove our use of IPI?

Reply:
Not yet but it's also defined in `pal/spi/iOS/UIKitSPI.h` - maybe I should use that instead?

Reply:
Looks like it's in `WebKit`, not `PAL` – but yeah let's just import "UIKitSPI.h".